core/meta/src/TListOfFunctions.cxx

```diff
@@ -259,7 +259,7 @@ TFunction *TListOfFunctions::Find(DeclId_t id) const
 /// Return (after creating it if necessary) the TMethod or TFunction
 /// describing the function corresponding to the Decl 'id'.
 
-TFunction *TListOfFunctions::Get(DeclId_t id)
+TFunction *TListOfFunctions::Get(DeclId_t id, bool verify)
```

Comment:
Can you add doc for `verify`? Can you maybe make the arg name a bit more explicit (what is being verified?) at the cost of a couple of characters? It might be "verifyIsMember" if that's what it is.